requirements/base.in

```diff
@@ -0,0 +1 @@
+# pip-compile --output-file base.txt base.in
```

Comment:
maybe these requirements/*.in and requirements/*.txt are not necessary? verify pls